pool_configs/examples/bzc.json

```diff
@@ -0,0 +1,124 @@
+{
+    "enabled": false,
+    "coin": "bzc.json",
+
+    "address": "",
+    "_comment_address": "a transparent address to send coinbase rewards to and to transfer to zAddress.",
+
+    "zAddress": "",
+    "_comment_zAddress": "a private address used to send coins to tAddress.",
+
+    "tAddress": "",
+    "_comment_tAddress": "transparent address used to send payments, make this a different address, otherwise payments will not send",
+
+    "invalidAddress":"",
+    "_comment_invalidAddress": "Invalid addresses will be converted to the above",
+
+    "walletInterval": 2.5,
+    "_comment_walletInterval": "Used to cache coin stats",
+
+    "rewardRecipients": {
+        "": 0.2
+    },
+
+    "tlsOptions": {
+        "enabled": false,
+        "serverKey": "",
+        "serverCert": "",
+        "ca": ""
+    },
+
+    "paymentProcessing": {
+        "enabled": true,
+        "minConf": 6,
+        "paymentMode": "prop",
+        "_comment_paymentMode": "prop, pplnt",
+        "paymentInterval": 120,
+        "minimumPayment": 0.05,
+        "maxBlocksPerPayment": 3,
+        "daemon": {
+            "host": "127.0.0.1",
+            "port": 8732,
+            "user": "rpcuser",
+            "password": "rpcpassword"
+        }
+    },
+
+    "ports": {
+        "1234": {
+            "tls":false,
+            "diff": 0.5,
+            "varDiff": {
+                "minDiff": 0.04,
+                "maxDiff": 16,
+                "targetTime": 15,
+                "retargetTime": 80,
+                "variancePercent": 30
+            }
+        },
+        "1234": {
+            "tls":false,
+            "diff": 16,
+            "varDiff": {
+                "minDiff": 6,
+                "maxDiff": 20,
+                "targetTime": 15,
+                "retargetTime": 80,
+                "variancePercent": 30
+            }
+        },
+        "1234": {
+            "tls":false,
+            "diff": 0.25,
+            "varDiff": {
+                "minDiff": 0.1,
+                "maxDiff": 10,
+                "targetTime": 15,
+                "retargetTime": 80,
+                "variancePercent": 30
+            }
+        },
+        "1234": {
+            "tls":false,
+            "diff": 4,
+            "varDiff": {
+                "minDiff": 0.5,
+                "maxDiff": 20,
+                "targetTime": 15,
+                "retargetTime": 80,
+                "variancePercent": 30
+            }
+        }
+    },
+
```

Comment:
I'm assuming you tested out these difficulties already and are recommended for this coin?